extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authorization/BasicRoleBasedAuthorizer.java

```diff
@@ -106,8 +114,16 @@ public Access authorize(AuthenticationResult authenticationResult, Resource reso
         }
       }
     }
-
-    return new Access(false);
+    LOG.info("User [%s] is not allowed to [%s] the resource [%s:%s]",
```

Comment:
```suggestion
    LOG.info("User [%s] does not have [%s] permission for the resource [%s:%s]",
```
nit: Making the log consistent with the error message